.github/workflows/build-test.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ macos-latest, ubuntu-20.04 ]
+        os: [ macos-latest, ubuntu-22.04 ]
```

Comment:
Consider using `ubuntu-latest` instead, we already do it for binds ([source](https://github.com/dashpay/bls-signatures/blob/0bb5c5b03249c463debb5cef5f7e52ee66f3aaab/.github/workflows/build-binds.yml#L25)) and it'd take care of `valgrind`-related failures ([build](https://github.com/dashpay/bls-signatures/actions/runs/13312724354/job/37179107055#step:8:18)) caused by Bug 452758 resolved in Valgrind 3.20 ([source](https://bugs.kde.org/show_bug.cgi?id=452758#c49)). `jammy` ships with 3.18 ([source](https://packages.ubuntu.com/jammy/valgrind)) while `noble` onwards ship with 3.22 ([source](https://packages.ubuntu.com/noble/valgrind)).

See https://github.com/kwvg/bls-signatures/actions/runs/13326187797 for 5552df305ffec8173429accff1478597a30bba26.